spec/spec_helper.rb

```diff
@@ -5,6 +5,11 @@
 require 'rubygems'
 require 'rspec'
 require 'lib/tools'
+require 'simplecov'
+SimpleCov.start
```

Comment:
Create a file in spec/support/codecov.rb with this. It’s for better organizing and it gets loaded once.

Reply:
Done! Trying my hands on rb for the first time.

Reply:
Look at CI failures, rubocop etc